tests/specs/graph/jsr/FastCheck_Overloads.txt

```diff
@@ -0,0 +1,89 @@
+# https://jsr.io/@scope/a/meta.json
+{"versions": { "1.0.0": {} } }
+
+# https://jsr.io/@scope/a/1.0.0_meta.json
+{ "exports": { ".": "./mod.ts" } }
+
+# https://jsr.io/@scope/a/1.0.0/mod.ts
+export class argsToArray {
+  argsToArray(..._args: string[]): void;
+  argsToArray(..._args: number[]): void;
+  argsToArray(..._args: unknown[]): void {}
+}
+
+function overloadSpread(...args: string[]);
+function overloadSpread(...args: number[]);
+export function overloadSpread(...args: unknown[]) {
+  return args;
+}
```

Comment:
Mixing non-exported overloads with an exported implementation is not valid in TS. See https://www.typescriptlang.org/play?#code/KYDwDg9gTgLgBAYwDYEMDOa4qgczQFQgEEooUBPOAbwCg4tcDjSKAKAOk4H1s8AuOGhhQAlgDscAbQC6ASgEA3CCIAmAbjoM8hEmXIduvNALEBXALYAjYFBny4S1RvpGdLfZ3Y9GA02IDWYhAA7mJ2isoq1AC+NLE0AGZ+CDAiEGJwEAo2SBAoKgDKYFDA+QbsRgJCohLhWGLkGkliKWkZWTl5hcWlKuWVcGZWNnUoDRqgkLBwza3pmdlQuflFJWWeA36BIWFyAmOUtPQlMKZQGUYa0UA

Reply:
whoops - you are right. fixed